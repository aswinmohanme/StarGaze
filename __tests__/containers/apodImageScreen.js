import 'react-native';
import React from 'react';
import ApodImageScreen from '../../app/containers/apodImageScreen';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <ApodImageScreen />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
