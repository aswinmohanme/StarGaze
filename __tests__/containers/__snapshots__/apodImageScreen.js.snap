// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "borderColor": "transparent",
      "borderRadius": 2,
      "borderWidth": 0,
      "flex": 1,
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Image
    animationOptions={Object {}}
    onLayout={[Function]}
    source={
      Object {
        "uri": "https://apod.nasa.gov/apod/image/1705/STSCI-HST-abell370_1797x2000.jpg",
      }
    }
    style={
      Object {
        "alignItems": "center",
        "flex": 0.8,
        "flexDirection": "column",
        "justifyContent": "center",
        "resizeMode": "cover",
      }
    }
    virtual={undefined}
  />
  <View
    animationOptions={Object {}}
    onLayout={[Function]}
    style={
      Object {
        "flex": 0.2,
      }
    }
    virtual={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      animationOptions={Object {}}
      ellipsizeMode="tail"
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#222222",
          "fontFamily": "Rubik-Regular",
          "fontSize": 20,
          "fontStyle": "normal",
          "fontWeight": "normal",
          "lineHeight": 25,
          "marginBottom": 15,
          "marginLeft": 15,
          "marginRight": 15,
          "marginTop": 15,
        }
      }
      styleName="md-gutter"
      virtual={undefined}
    >
      Awesome Title
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      animationOptions={Object {}}
      ellipsizeMode="tail"
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#666666",
          "fontFamily": "Rubik-Regular",
          "fontSize": 16,
          "fontStyle": "normal",
          "fontWeight": "normal",
          "marginLeft": 15,
        }
      }
      styleName="md-gutter-left"
      virtual={undefined}
    >
      Awesome Description
    </Text>
  </View>
</View>
`;
